pyaedt/application/Variables.py

```diff
@@ -452,7 +496,7 @@ class VariableManager(object):
      'Var3': <pyaedt.application.Variables.Expression at 0x2661f34cb48>,
      '$PrjVar1': <pyaedt.application.Variables.Expression at 0x2661f34cc48>}
 
-    >>> # get a dictionary of just the design variables
+    >>> # Get a dictionary of just the design variables
     >>> v.design_variables
 
```

Comment:
```suggestion
```
There cannot be a space after the return value of a call and the return itself.  Otherwise, this gets rendered as plain text.

Reply:
Please fix the examples above and below this as well.